cglib/src/main/java/net/sf/cglib/core/CodeEmitter.java

```diff
@@ -806,6 +806,7 @@ public void zero_or_null(Type type) {
                 break;
             case Type.VOID:
                 aconst_null();
+                // fall through
```

Comment:
-1
It looks like a true bug, not a false positive.

Well, it is hard to tell what sense makes 0 or null for VOID type, however `fall through` creates both of them, and it apparently is not the right thing.

Both usages of `zero_or_null` ensure the argument is not VOID, so it clarifies why the error was not observed earlier.

Reply:
Even better.  It's nice when static analysis finds real bugs.  I'm not sure what the fix is; I was only trying to appease error prone when compiling cglib with bazel.